### PR TITLE
ENG-6137: Fix bug where interactive setups weren't working

### PR DESCRIPTION
**Pull request by watsonian**
This fixes a bug introduced in #374 that caused interactive invocations of `doppler setup` to operate as no-ops.

### CODE REVIEW

File: pkg/controllers/repo_config.go
Comment (mrasnake-doppler) [inline pkg/controllers/repo_config.go:76]: Just a note, but from a Go standards perspective I don't think I like what we are doing with this `Error` object. But I get that it would require a greater refactor then the scope of this issue. Something I will look into when I have cycles for a redesign. 

Other then that I will approve the Go code, but recommend you also wait for an approval from someone with a deeper understanding of the business logic.

**Review by mrasnake-doppler — APPROVED** [review]
Just a note, but from a Go standards perspective I don't think I like what we are doing with this Error object. But I get that it would require a greater refactor then the scope of this issue. Something I will look into when I have cycles for a redesign.

Other then that I will approve the Go code, but recommend you also wait for an approval from someone with a deeper understanding of the business logic.

**Comment by watsonian** [comment]
We've decided to roll back to prior to this change so we have more time to review things to make sure there aren't any other issues surrounding this. I'm going to close this PR for now.

**Pull request closed without merging**
